TeamAmaze/AmazeFileManager
# F-Droid friendly codebase

## Description

To make `gradlew clean spotlessApply spotlessCheck jacocoTestFdroidDebugUnitTestReport assembleDebug` build workflow also work seamlessly,

- Moved RAR related tests to `testPlay` directory
- `CompressedHelperCallableTestSuite` for different flavours
- `ExtractorTestSuite` for different flavours
- Added build flavour checking in `CompressedHelperTest` and `ExtractServiceTest`

 
#### Manual tests

By doing a `./gradlew clean spotlessApply spotlessCheck jacocoTestFdroidDebugUnitTestReport clean spotlessApply spotlessCheck jacocoTestPlayDebugUnitTestReport assembleDebug --stacktrace` that can finish without any errors. Also check RAR related tests in `play` flavour are run, not skipped.

#### Build tasks success  
Successfully running following tasks on local:
